app/models/document.rb

```diff
@@ -47,6 +47,20 @@ def promoted
 
   attr_reader :link, :rummager_document, :finder, :description
 
+  def is_mainstream_content?
+    %w(completed_transaction
+       local_transaction
+       calculator
+       smart_answer
+       simple_smart_answer
+       place
+       licence
+       step by step
+       transaction
+       answer
+       guide).include?(@document_type)
```

Comment:
Is there anywhere this is stored centrally? Like Govuk Document Types?

Reply:
couldn't find it anywhere myself

Reply:
Should step by step be underscored?

Reply:
you're right. serves me for copy-pasting from trello! good spot